### ISSUE TITLE
Can you add environment variables to a running container?

### ISSUE THREAD

**Issue by ISuckAtPython** [moby/moby#37884]
**Steps to reproduce the issue:**
1. Run a container 
2. Commands can be ran with one time environment variables, but permanent ones cannot be added

**Output of `docker version`:**

```
Version: 17.03.2-ce
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
CoreOS running an etcd cluster

**Comment by thaJeztah** [comment]
No, changing environment variables of the container is not currently supported; effectively, a container is considered to be immutable after it's created (baring some resource-settings that can be changed with `docker container update`)
